ets-ecole-de-technologie-superieure.csl

```diff
@@ -122,21 +122,28 @@
     <text variable="issue" prefix=" "/>
   </macro>
   <macro name="access">
-    <group delimiter=". ">
-      <text variable="URL" prefix="&lt;" suffix="&gt;"/>
-      <choose>
-        <if variable="URL" match="any">
-          <group delimiter=" ">
-            <text value="Consulté le"/>
-            <date variable="accessed">
-              <date-part name="day" suffix=" "/>
-              <date-part name="month" suffix=" "/>
-              <date-part name="year"/>
-            </date>
-          </group>
-        </if>
-      </choose>
-    </group>
+		<choose>
+			<if variable="DOI" match="any">
+				<group delimiter=". ">
+					<text variable="DOI" prefix="&lt;http://dx.doi.org/" suffix="&gt;"/>
```

Comment:
this is fine, but if you have any input on the style guidelines, doi.org actually prefers `http://doi.org`, i.e. without the dx. part, now, so I'd encourage you to use that (though we'll accept either)